src/App.js

```diff
@@ -1,16 +1,57 @@
 import React from 'react';
 import './App.css';
+import { useState } from 'react';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+
 
 const App = () => {
+  const [messageData, setMessageData] = useState(chatMessages);
+
+  const updateMessageData = updatedMessage => {
+    const messages = messageData.map(message => {
+      if (message.id === updatedMessage.id) {
+        return updatedMessage;
+      } else {
+        return message;
+      }
+    });
+    setMessageData(messages);
+  };
+
+  const assignUniqueIdToSender = returnedMessage => {
+    if (returnedMessage.sender === 'Estragon') {
+      return 1
+    } else {
+      return 0
+    }
+  };
+
+
+  const calcTotalLikes = chatMessages => {
+    return chatMessages.reduce((total, message)=> {
+      return total + message.liked
+    }, 0);
+  };
+
+  const totalLikeCount = calcTotalLikes(messageData);
+
   return (
     <div id="App">
       <header>
-        <h1>Application title</h1>
+        <h1>Chat between Vladimir and Estragon</h1>
+        <section>
+          <h2>{totalLikeCount} ❤️s</h2>
+        </section>
       </header>
       <main>
-        {/* Wave 01: Render one ChatEntry component
-        Wave 02: Render ChatLog component */}
+        {/*Wave 02: Render ChatLog component */}
+        {/* components are the tags and props are the attributes */}
```

Comment:
In industry, when you open a PR your code should be in a state as if it were ready to be merged into production code. Therefore we want to get rid of unused comments and imports to keep the file from getting cluttered.